src/managementgroups/azext_managementgroups/_help.py

```diff
@@ -0,0 +1,157 @@
+    long-summary: Update an existing management group.
+    parameters:
+        - name: --group-name --name -n
+          type: string
+          short-summary: Name of the management group.
+        - name: --display-name -d
+          type: string
+          short-summary: Updates the display name of the management group. If null, no change is made.
+        - name: --parent-id -p
+          type: string
+          short-summary: Update the parent of the management group. A fully qualified id is required. If null, no change is made.
+    examples:
+        - name: Update an existing management group with a specific display name.
+          text: >
+             az managementgroups group update --group-name GroupName --display-name DisplayName
+        - name: Update an existing management group with a specific parent id.
+          text: >
+             az managementgroups group update --group-name GroupName --parent-id ParentId
+        - name: Update an existing management group with a specific display name and parent id.
+          text: >
+             az managementgroups group update --group-name GroupName --display-name DisplayName --parent-id ParentId
+"""
+
+helps['account management-group delete'] = """
+    type: command
+    short-summary: Remove an existing management group.
+    long-summary: Remove an existing management group.
+    parameters:
+        - name: --group-name --name -n
+          type: string
+          short-summary: Name of the management group.
+    examples:
+        - name: Remove an existing management group
+          text: >
+             az managementgroups group remove --group-name GroupName
+"""
+
+helps['account management-group subscription add'] = """
+    type: command
+    short-summary: Add a subscription to a management group.
+    long-summary: Add a subscription to a management group.
+    parameters:
+        - name: --group-name --name -n
+          type: string
+          short-summary: Name of the management group.
+        - name: --subscription
+          type: string
+          short-summary: Subscription Id or Name
+    examples:
+        - name: Add a subscription to a management group.
+          text: >
+             az managementgroups group new --group-name GroupName --subscription <subscription>
+"""
+
+helps['account management-group subscription remove'] = """
+    type: command
+    short-summary: Remove an existing subscription from a management group.
+    long-summary: Remove an existing subscription from a management group.
+    parameters:
+        - name: --group-name --name -n
+          type: string
+          short-summary: Name of the management group.
+        - name: --subscription
+          type: string
+          short-summary: Subscription Id or Name
+    examples:
+        - name: Remove an existing subscription from a management group.
+          text: >
+             az managementgroups group remove --group-name GroupName --subscription <subscription>
```

Comment:
Same with ` <subscription>` -> `Subscription`

Reply:
Missed that. Thanks!